docs/metrics.md

```diff
@@ -0,0 +1,86 @@
+# Metrics provided by KMS plugin for Key Vault
+
+This project uses [opentelemetry](https://opentelemetry.io/) for reporting metrics. Please refer it's status [here](https://github.com/open-telemetry/opentelemetry-go#project-status). Prometheus is the only exporter that's currently supported.
+
+## List of metrics provided by the kms plugin
+
+| Metric                          | Description                                                               | Tags                                                                              |
+| ------------------------------- | ------------------------------------------------------------------------- | --------------------------------------------------------------------------------- |
+| total_request                   | Total number of requests                                                  | `status=success OR error`<br>`operation=encrypt OR decrypt`                           |
```

Comment:
Do we also want to add the optional `error_message` label here?

Reply:
updated entire metrics section